Resources/Prototypes/Adventure/_CyberneticGenesis/LilaKebab/meal_recipes.yml

```
@@ -0,0 +1,17 @@
+#Pizzas TODO: contruction graph based pizza
+- type: microwaveMealRecipe
+  id: RecipeLilaKebabShaurma
+  name: шаурма ли ла кебаб
+  result: FoodMealLilaKebab
+  time: 30
+  group: Savory
+  reagents:
+    BbqSauce: 5
+    Ketchup: 5
+  solids:
+    FoodDoughFlat: 3
+    FoodTomato: 1

```

```suggestion
    FoodTomato: 1
```
Помидор же вроде тоже можно на слайсы порезать, или нет?

---

насчёт помидорок не помню, мне выдавало в спавн меню только его целую часть, так что вряд-ли

---

Слайсов помидоров нету, по крайней мере на моей версии локалки, а её я делал вроде в начале августа